testmod/buid.gradle

```diff
@@ -0,0 +1,5 @@
+archivesBaseName = "fabric-testmod"
```

Comment:
Incorrect filename here?

Reply:
Yep, can you tell I didnt really test it :)